CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
+### Changed
+- **BREAKING**: Rename package from `ethjs-util` to `@metamask/ethjs-util`
+- **BREAKING**: Deprecate nodejs < v8.17, npm < v6
```

Comment:
Nit: Maybe worth using stronger wording here?

``` suggestion
- **BREAKING**: Bump minimum required version of Node to 8.17.0
```

The NPM change seems development-only, but in case you want to include it anyway:

``` suggestion
- **BREAKING**: Bump minimum required version of Node to 8.17
- **BREAKING**: Bump minimum required version of NPM to 6
```